docs/unleash_a-b_testing.md

```diff
@@ -112,6 +112,26 @@ function MyComponent() {
 
 The number of segment events sent for an experiment is [throttled](https://github.com/artsy/force/blob/main/src/v2/System/useFeatureFlag.tsx#L65) to one event per user, per variant view.
 
+### Accessing flags on the server
+
+If needing to access a feature flag on the server before rendering any components, all feature flags are located within local sharify data.
+
+#### Example
+
+```tsx
+const routes = [
+  {
+    path: "/foo",
+    onServerSideRender: ({ res }) => {
+      if (!res.locals.sd.FEATURE_FLAGS.myFlag) {
```

Comment:
@mc-jones - we might want to expand this a bit, but this should communicate the point (i think)